src/main/java/com/TheAccountant/model/payment/Payment.java

```diff
@@ -0,0 +1,109 @@
+package com.TheAccountant.model.payment;
+
+import com.TheAccountant.model.user.AppUser;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+
+/**
+ * Created by Florin on 8/27/2017.
+ */
+@Entity
+@Table(name = "PAYMENT")
+public class Payment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @ManyToOne(fetch = FetchType.EAGER, optional = false)
```

Comment:
Why ManyToOne? Because of failure/refunded situations?

Reply:
Yes, there can be situations in which the transaction is not approved or is refunded.